README.md

```diff
@@ -1,148 +1,181 @@
 
-Requirements
-------------
+## Requirements
 
 An Ansible 2.3 or higher installation.
 
-Dependencies
-------------
+## Dependencies
 
 None.
 
-Role Variables
---------------
+## Role Variables
 
 Available variables are listed below, along with default values (see `defaults/main.yml`):
 
-    dnsdist_install_repo: "" 
+```yaml
+dnsdist_install_repo: ""
+```
 
 By default dnsdist is installed from the os default repositories.
 You can install dnsdist from the official PowerDNS repository overriding
 the `dnsdist_install_repo` variable value as follows
 (for the complete list of pre-defined repos see `vars/main.yml`):
 
-    # Install dnsdist from the master branch
-    - hosts: pdns-dnsdists
-      roles:
-      - { role: PowerDNS.dnsdist,
-          dnsdist_install_repo: "{{ dnsdist_powerdns_repo_master }}"
-
-    # Install dnsdist 1.3.x
-    - hosts: pdns-dnsdists
-      roles:
-      - { role: PowerDNS.dnsdist,
-          dnsdist_install_repo: "{{ dnsdist_powerdns_repo_13 }}"
-
-The roles also supports custom repositories
-
-    - hosts: all
-      vars:
-        dnsdist_install_repo:
-          name: "dnsdist" # the repository name
-          apt_repo_origin: "my.repo.com"  # used to pin dnsdist to the provided repository
-          apt_repo: "deb http://my.repo.com/{{ ansible_distribution | lower }} {{ ansible_distribution_release | lower }}/dnsdist main"
-          gpg_key: "http://my.repo.com/MYREPOGPGPUBKEY.asc" # repository public GPG key
-          gpg_key_id: "MYREPOGPGPUBKEYID" # to avoid to reimport the key each time the role is executed
-          yum_repo_baseurl: "http://my.repo.com/centos/$basearch/$releasever/dnsdist"
-          yum_debug_symbols_repo_baseurl: "http://repo.powerdns.com/centos/$basearch/$releasever/dnsdist/debug"
-      roles:
-      - { role: PowerDNS.dnsdist }
+```yaml
+# Install dnsdist from the master branch
+- hosts: pdns-dnsdists
+  roles:
+  - { role: PowerDNS.dnsdist,
+      dnsdist_install_repo: "{{ dnsdist_powerdns_repo_master }}"
+
+# Install dnsdist 1.3.x
+- hosts: pdns-dnsdists
+  roles:
+  - { role: PowerDNS.dnsdist,
+      dnsdist_install_repo: "{{ dnsdist_powerdns_repo_13 }}"
+```
+
+The role also supports custom repositories
+
+```yaml
+- hosts: all
+  vars:
+    dnsdist_install_repo:
+      name: "dnsdist" # the repository name
+      apt_repo_origin: "my.repo.com"  # used to pin dnsdist to the provided repository
+      apt_repo: "deb http://my.repo.com/{{ ansible_distribution | lower }} {{ ansible_distribution_release | lower }}/dnsdist main"
+      gpg_key: "http://my.repo.com/MYREPOGPGPUBKEY.asc" # repository public GPG key
+      gpg_key_id: "MYREPOGPGPUBKEYID" # to avoid to reimport the key each time the role is executed
+      yum_repo_baseurl: "http://my.repo.com/centos/$basearch/$releasever/dnsdist"
+      yum_debug_symbols_repo_baseurl: "http://repo.powerdns.com/centos/$basearch/$releasever/dnsdist/debug"
```

Comment:
you might wanna set this to the custom domain as well.

Reply:
Right!